pytorch_caney/pipelines/pretraining/simmim.py

```diff
@@ -29,7 +29,7 @@
 
 def parse_args():
     parser = argparse.ArgumentParser(
-        'pytorch-caney impletmentation of SimMiM pre-training script',
+        'pytorch-caney impletmentation of MiM pre-training script',
```

Comment:
implementation instead of impletmentation

Reply:
so does the name of the script need to be changed to "mim.py"?

Reply:
Fixed spelling in https://github.com/nasa-nccs-hpda/pytorch-caney/pull/22/commits/41b53be4f3b47e52a052d61012e2d881b1d12106

Reply:
Changed name of script in https://github.com/nasa-nccs-hpda/pytorch-caney/pull/22/commits/1632f189ded29934679cbdbd694210a067654e47